docs/guides/key-manager/execute-relay-transactions.md

```diff
@@ -130,7 +130,7 @@ const keyManager = new web3.eth.Contract(KeyManagerContract.abi, keyManagerAddre
 ```typescript title="Contract instances"
 const universalProfile = new ethers.Contract(universalProfileAddress, UniversalProfileContract.abi);
 
-const keyManagerAddress = await universalProfile.owner();
+const keyManagerAddress = await universalProfile.connect(controllerAccount).owner();
```

Comment:
is the `.connect(...)` needed here? This is just a getter function. Should not be necessary.

Reply:
getting error without because it doesn't know the provider